pipeline_dp/dp_engine.py

```diff
@@ -227,38 +227,39 @@ def sample_unique_elements_fn(pid_and_pks):
 
             return ((pid, pk) for pk in sampled_elements)
 
-        col = self._ops.flat_map(col, sample_unique_elements_fn,
-                                 "Sample cross-partition contributions")
+        col = self._backend.flat_map(col, sample_unique_elements_fn,
+                                     "Sample cross-partition contributions")
         # col : (privacy_id, partition_key)
 
         # A compound accumulator without any child accumulators is used to calculate the raw privacy ID count.
         compound_combiner = CompoundCombiner([])
-        col = self._ops.map_tuple(
+        col = self._backend.map_tuple(
             col, lambda pid, pk: (pk, compound_combiner.create_accumulator([])),
             "Drop privacy id and add accumulator")
         # col : (partition_key, accumulator)
 
-        col = self._ops.combine_accumulators_per_key(
+        col = self._backend.combine_accumulators_per_key(
             col, compound_combiner, "Combine accumulators per partition key")
         # col : (partition_key, accumulator)
 
         col = self._select_private_partitions_internal(
             col, max_partitions_contributed)
-        col = self._ops.keys(col, "Drop accumulators, keep only partition keys")
+        col = self._backend.keys(col,
+                                 "Drop accumulators, keep only partition keys")
 
         return col
 
     def _drop_not_public_partitions(self, col, public_partitions,
                                     data_extractors: DataExtractors):
-        """Drops partitions in `col` which are not in `public_partitions`."""
-        col = self._ops.map(
+        """Drbackend partitions in `col` which are not in `public_partitions`."""
```

Comment:
"Drops". Find-and-replace can be tricky sometimes. :)

Reply:
Thanks! I should have used whole words replacement :)